admin-ui/plugins/auth-server/components/AuthN/index.js

```diff
@@ -1,42 +1,40 @@
 import React from 'react'
-import AuthNListPage from './AuthNListPage'
-import { useTranslation } from 'react-i18next'
 import GluuTabs from 'Routes/Apps/Gluu/GluuTabs'
 import { Card } from 'Components'
 import applicationStyle from 'Routes/Apps/Gluu/styles/applicationstyle'
-
 import AgamaListPage from '../Agama/AgamaListPage'
 import AliasesListPage from '../Agama/AgamaAliasListPage'
+import ScriptsListPage from './ScriptsListPage'
+import LdapListingPage from './LdapListingPage'
+import Basic from './Basic'
+import DefaultAcr from './DefaultAcr'
 
 function AuthNPage() {
-  const { t } = useTranslation()
-
   const tabNames = [
-    {
-      name: t('menus.builtIn'),
-      path: '',
-    },
-    { name: t('menus.acrs'), path: '' },
-    {
-      name: t('menus.aliases'),
-      path: '',
-    },
-    {
-      name: t('menus.agama_flows'),
-      path: '',
-    },
+    { name: 'default acr', path: '' },
+    { name: 'built in', path: '' },
+    { name: 'ldap servers', path: '' },
+    { name: 'scripts', path: '' },
+    { name: 'aliases', path: '' },
+    { name: 'agama flows', path: '' },
   ]
 
   const tabToShow = (tabName) => {
     switch (tabName) {
-      case t('menus.builtIn'):
-        return <AuthNListPage isBuiltIn={true} />
-      case t('menus.acrs'):
-        return <AuthNListPage />
-      case t('menus.aliases'):
+      case 'default acr':
+        return <DefaultAcr />
+      case 'built in':
+        return <Basic />
+      case 'ldap servers':
+        return <LdapListingPage />
+      case 'scripts':
+        return <ScriptsListPage />
+      case 'aliases':
         return <AliasesListPage />
-      case t('menus.agama_flows'):
+      case 'agama flows':
         return <AgamaListPage />
+      default:
+        return <DefaultAcr />
```

Comment:
Switch tab resolution to use stable IDs or translation keys  
In AuthN/index.js, avoid switching on hard-coded labels: either populate each `tabNames` entry’s `path` (or add an explicit `id`) and have `tabToShow` switch on that field, or import your i18n hook (`useTranslation`) and wrap both the `name` in `tabNames` and the switch cases with `t('…')` to keep UI text and logic in sync.